src/granite_orm/querying.cr

```diff
@@ -92,6 +92,11 @@ module Granite::ORM::Querying
     end
   end
 
+  # count returns a count of all the records
+  def count : Int64
```

Comment:
No objection here but why did you choose `Int64`?

Reply:
I was thinking about handling large tables like 10 billion data.

However, in this case, special queries such as `CAST AS BIGINT` may be required. In fact, there are no rare cases beyond `Int32` at this moment.

So I'd like to modify it to use `Int32`. Thought?

Reply:
I have no preference. I think Postgres might return a different type depending on how much data comes back, so that may be an issue.